libs/partners/ai21/langchain_ai21/chat_models.py

```diff
@@ -35,6 +35,8 @@ class ChatAI21(BaseChatModel, AI21Base):
         You can view the options at https://github.com/AI21Labs/ai21-python?tab=readme-ov-file#model-types"""
     num_results: int = 1
     """The number of responses to generate for a given prompt."""
+    stop: Optional[List[str]] = None
```

Comment:
same comment about making this stop_sequences and alias stop

Reply:
missed your comments, will add in another PR